### PR TITLE
Use first alphabetical match for language

### LINKED ISSUE #10: lang-tags-from-hb.py does not match HarfBuzz’s disambiguation strategy
lang-tags-from-hb.py disambiguates language system tags that correspond to multiple BCP 47 tags by parsing `hb_ot_ambiguous_tag_to_language`. However, this does not catch ambiguous language system tags not handled by that function. For example, 'IWR ' should be converted to he, and yet it current is:
https://github.com/Wakamai-Fondue/wakamai-fondue-engine/blob/c181dd619375325f27897f9d81c395bd473f016c/src/tools/ot-to-html-lang.js#L878-L881

HarfBuzz’s disambiguation strategy is to consult `hb_ot_ambiguous_tag_to_language` then, if there was no match, pick the alphabetically first match in `ot_languages`. Wakamai Fondue’s strategy is to consult `hb_ot_ambiguous_tag_to_language` then, if there was no match, pick the alphabetically _last_ match in `ot_languages`. (If iw were the recommended code for Hebrew and he were the deprecated code, then HarfBuzz would have to handle Hebrew explicitly, but because the recommended code happens to alphabetically precede the other, it doesn’t.)

### PR DESCRIPTION
**Pull request by RoelN**
This will match HarfBuzz' approach, whereas before we were using
the *last* alphabetical match, resulting in errors.

Fixes #10

Thank you @dscorbett for bringing this to my attention!

### CODE REVIEW

**Comment by dscorbett** [comment]
You might also want to strip the suffixes “[family]” and “(retired code)”.

**Comment by RoelN** [comment]
@dscorbett Good catch, thanks again!!

File: tools/lang-tags-from-hb.py [hunk 1]
```diff
@@ -82,6 +100,10 @@ def extract_languages(content):
 
             langdict[am_lang_ot] = {"html": am_lang_bcp, "name": am_lang_name}
 
+    for lr_lang in last_resort_langdict:
```
Comment (pascalw) [inline tools/lang-tags-from-hb.py:103]: Is it really necessary to keep track of the `last_resort_langdict` and only at the very end of processing replace items in the in the `langdict`? IMO the code would be easier to understand if it built up just once list (`langdict`). It seems that the logic for using a `last_resort` option is already present at line 75-78.

**Review by pascalw — APPROVED** [review]

File: tools/lang-tags-from-hb.py (hunk 1 above)
Comment (RoelN) [reply tools/lang-tags-from-hb.py:103]: They truly are last resort languages. I think this code is indeed somewhat clunky, but it does catch the logic of choosing a lannguage. Might revisit layer, thanks for spotting this though!

**Pull request merged**